torchrl/data/utils.py

```diff
@@ -42,6 +43,36 @@
 INDEX_TYPING = Union[None, int, slice, str, Tensor, List[Any], Tuple[Any, ...]]
 
 
+def dense_stack_tds(
```

Comment:
I'll write some tests and examples for it.

I am down to move it to tensodict, I put it here because I thought we said we wanted the utis to be here for the deprecation move